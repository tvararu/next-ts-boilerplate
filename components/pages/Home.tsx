const Home = () => <p className="uppercase">Hello, world</p>;

export default Home;
