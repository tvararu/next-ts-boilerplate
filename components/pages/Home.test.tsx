import renderer from "react-test-renderer";
import Home from "./Home";

describe("Home", () => {
  it("renders correctly", () => {
    const component = renderer.create(<Home />);
    const tree = component.toJSON();

    expect(tree).toMatchSnapshot();
  });
});
