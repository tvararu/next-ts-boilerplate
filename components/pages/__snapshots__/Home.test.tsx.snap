// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders correctly 1`] = `
<p
  className="uppercase"
>
  Hello, world
</p>
`;
